src/applications/gi/components/vet-tec/VetTecContactInformation.jsx

```diff
@@ -1,41 +1,202 @@
+      return 0;
+    })
+    .slice(0, 2);
+  return rankedList;
+};
 
 export const VetTecContactInformation = ({ institution }) => (
-  <div className="additional-information row vads-u-margin-y--1">
-    <div className="usa-width-one-half medium-6 columns">
-      <div className="physical-address usa-width-one-whole">
-        <h3>Physical address</h3>
-        <div>
-          {institution.physicalAddress1 && (
-            <div>{institution.physicalAddress1}</div>
-          )}
-          {institution.physicalAddress2 && (
-            <div>{institution.physicalAddress2}</div>
-          )}
-          {institution.physicalAddress3 && (
-            <div>{institution.physicalAddress3}</div>
-          )}
+  <div>
+    <div className="additional-information row vads-u-margin-y--1">
+      <div className="usa-width-one-half medium-6 columns">
+        <div className="physical-address usa-width-one-whole">
+          <h3>Physical address</h3>
           <div>
-            {institution.physicalCity}, {institution.physicalState}{' '}
-            {institution.physicalZip}
+            {institution.physicalAddress1 && (
+              <div>{institution.physicalAddress1}</div>
+            )}
+            {institution.physicalAddress2 && (
+              <div>{institution.physicalAddress2}</div>
+            )}
+            {institution.physicalAddress3 && (
+              <div>{institution.physicalAddress3}</div>
+            )}
+            <div>
+              {institution.physicalCity}, {institution.physicalState}{' '}
+              {institution.physicalZip}
+            </div>
           </div>
         </div>
       </div>
-    </div>
-    <div className="usa-width-one-half medium-6 columns">
-      <div className="mailing-address usa-width-one-whole">
-        <h3>Mailing address</h3>
-        <div>
-          {institution.address1 && <div>{institution.address1}</div>}
-          {institution.address2 && <div>{institution.address2}</div>}
-          {institution.address3 && <div>{institution.address3}</div>}
+      <div className="usa-width-one-half medium-6 columns">
+        <div className="mailing-address usa-width-one-whole">
+          <h3>Mailing address</h3>
           <div>
-            {institution.city}, {institution.state} {institution.zip}
+            {institution.address1 && <div>{institution.address1}</div>}
+            {institution.address2 && <div>{institution.address2}</div>}
+            {institution.address3 && <div>{institution.address3}</div>}
+            <div>
+              {institution.city}, {institution.state} {institution.zip}
+            </div>
           </div>
         </div>
       </div>
     </div>
+    {/* Production flag for 19534 */}
+    {!environment.isProduction() &&
+      institution.schoolCertifyingOfficials.length > 0 && (
+        <div className="additional-information row vads-u-margin-y--4">
+          <div className="usa-width-one-half medium-6 columns">
```

Comment:
Got it, that makes sense now that the sorting function is no longer required

Reply:
Doesn't have to be a looping function, just a function that creates these since the only difference I saw was the `<h3>` and which element of the array is being used

Reply:
@zurbergram The latest commit should have the requested changes for this PR